examples/relative_free_energy.py

```diff
@@ -59,7 +59,7 @@ def run_pair(mol_a, mol_b, core, forcefield, n_frames, protein_path, seed):
         fh.write(complex_res.overlap_detail_png)
     write_trajectory_as_pdb(mol_a, mol_b, core, complex_res.frames, complex_top, "complex_traj")
 
-    print(f"complex dG: {np.sum(solvent_res.all_dGs):.3f} +- {np.linalg.norm(solvent_res.all_errs):.3f} kJ/mol")
+    print(f"complex dG: {np.sum(complex_res.all_dGs):.3f} +- {np.linalg.norm(complex_res.all_errs):.3f} kJ/mol")
```

Comment:
Agreed -- this seems like something `SimulationResult` should possibly be in charge of reporting https://github.com/proteneer/timemachine/blob/a0757b4cd72482226fd7724138c4f92a4f6ba2a2/timemachine/fe/free_energy.py#L48 ?

(Just proposing to fix this typo since it recently caused confusion, but maybe addressing deeper causes would be appropriate.)

Reply:
I was just doing a drive by review from the perspective of why we missed this.